### ISSUE TITLE
Decode GlobalLootModifier with Porting Lib?

### ISSUE THREAD

**Issue by suerion** [KekeCreations/Arts-and-Crafts#27]
**Describe the bug**
In our Server Console, there are some Porting lib Loot Modifier decoding Errors. I used the newest version from the Mod.

```
[15:29:02] [main/WARN]: Could not decode GlobalLootModifier with json id arts_and_crafts:ruined_pottery_sherd_from_archaeology_common - error: Unknown registry key in ResourceKey[minecraft:root / porting_lib:global_loot_modifier_serializers]: arts_and_crafts:modify_item
[15:29:02] [main/WARN]: Could not decode GlobalLootModifier with json id arts_and_crafts:cork_hanging_sign_from_archaeology_common - error: Unknown registry key in ResourceKey[minecraft:root / porting_lib:global_loot_modifier_serializers]: arts_and_crafts:modify_item
[15:29:02] [main/WARN]: Could not decode GlobalLootModifier with json id arts_and_crafts:pottery_sherd_from_archaeology_common - error: Unknown registry key in ResourceKey[minecraft:root / porting_lib:global_loot_modifier_serializers]: arts_and_crafts:modify_item
[15:29:02] [main/WARN]: Could not decode GlobalLootModifier with json id arts_and_crafts:bleachdew_from_archaeology_common - error: Unknown registry key in ResourceKey[minecraft:root / porting_lib:global_loot_modifier_serializers]: arts_and_crafts:modify_item
[15:29:02] [main/WARN]: Could not decode GlobalLootModifier with json id arts_and_crafts:finale_pottery_sherd_from_archaeology_rare - error: Unknown registry key in ResourceKey[minecraft:root / porting_lib:global_loot_modifier_serializers]: arts_and_crafts:modify_item
[15:29:02] [main/WARN]: Could not decode GlobalLootModifier with json id arts_and_crafts:ruined_pottery_sherd_from_archaeology_rare - error: Unknown registry key in ResourceKey[minecraft:root / porting_lib:global_loot_modifier_serializers]: arts_and_crafts:modify_item
[15:29:02] [main/WARN]: Could not decode GlobalLootModifier with json id arts_and_crafts:roll_pottery_sherd_from_archaeology_rare - error: Unknown registry key in ResourceKey[minecraft:root / porting_lib:global_loot_modifier_serializers]: arts_and_crafts:modify_item
[15:29:02] [main/WARN]: Could not decode GlobalLootModifier with json id arts_and_crafts:gateway_pottery_sherd_from_archaeology_rare - error: Unknown registry key in ResourceKey[minecraft:root / porting_lib:global_loot_modifier_serializers]: arts_and_crafts:modify_item
[15:29:02] [main/WARN]: Could not decode GlobalLootModifier with json id arts_and_crafts:lotus_pistils_from_sniffer - error: Unknown registry key in ResourceKey[minecraft:root / porting_lib:global_loot_modifier_serializers]: arts_and_crafts:modify_item
```

**To Reproduce**
1. Start Server with Arts and Crafts
2. Using a Hugh Modpack,
3. Get Warnings on server Console

**Game Environment**
Please provide details about the game environment, either by crash report/logs or by detailing it below. Try to reproduce the issue without irrelevant mods first. Otherwise, include a complete list of mods or the name and version of the modpack you're using.
 - Arts and Crafts: 1.1.9
 - Forge/Fabric Api: Fabric  1.21.1 on 0.16.5 
 - https://mclo.gs/iKlXez6
 - FarmersDelight Mods with PortingLib

**Comment by KekeCreations** [comment]
This issue was fixed in V1.2.0, thank you for reporting bugs!

**Comment by suerion** [comment]
Thanks also :)
